.github/workflows/build.yml

```diff
@@ -25,8 +25,8 @@ jobs:
         - destination: 'name=iPhone 8,OS=13.7'
           scheme: AdyenUIHost
     steps:
-    - uses: actions/checkout@v2
-    - uses: n1hility/cancel-previous-runs@v2
+    - uses: actions/checkout@v3
```

Comment:
@Adyen/checkout-ios The pipeline fails for v4 branch. Which is why i tried this but this too also fails. Any suggestions here?